Shared/SentryIntegration.swift

```diff
@@ -96,11 +102,15 @@ public class SentryIntegration {
         Client.shared?.snapshotStacktrace {
             let event = Event(level: severity)
             event.message = message
-            event.tags = ["event": tag]
+            event.tags = ["tag": tag]
 
             Client.shared?.appendStacktrace(to: event)
             event.debugMeta = nil
             Client.shared?.send(event: event, completion: completion)
         }
     }
+
+    public func setAttributes(_ attributes: [String : Any]) {
```

Comment:
It would also override existing values which is why I named it that. It doesn't matter much to me though.